onnxscript/function_libs/torch_lib/graph_building.py

```diff
@@ -130,13 +134,21 @@ def shape(self, shape: Tuple[int | None, ...]):
     @property
     def dtype(self):
```

Comment:
nit: is it possible to annotate return type?

Reply:
Done